packages/mdast/mail-templates/src/shared/components/Datawrapper.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { EdgeToEdgeFigure, Figure } from './Figure'
+
+// We generate the following images in Datawrapper upon publishing:
+//  1. https://datawrapper.dwcdn.net/DW123ID/mail.png -> 450px wide
+//  2. https://datawrapper.dwcdn.net/DW123ID/mailplain.png -> 450px wide, no header or footer
+//  3. https://datawrapper.dwcdn.net/DW123ID/edgetoedge.png -> 1280px wide
+//  4. https://datawrapper.dwcdn.net/DW123ID/edgetoedgeplain.png -> 1280px wide, no header or footer
+
+const Chart = ({ src, alt, width }) => (
```

Comment:
Where does the alt prop come from? It's part of the chart metadata but we don't load that anywhere.

Reply:
ah, we removed it from the config modal. what's your opinion? re-add it there or delete it here? i think it's kinda nice to have but i just saw that DW also has a Beschriebung field, so maybe let's just delete it everywhere?

Reply:
removed alt prop